static/app/components/radio.tsx

```diff
@@ -38,8 +38,8 @@ const Radio = styled('input')<Props>`
   /* TODO(bootstrap): Our bootstrap CSS adds this, we can remove when we remove that */
   margin: 0 !important;
 
-  &:focus,
-  &.focus-visible {
+  &&:focus,
+  &&.focus-visible {
     outline: none;
```

Comment:
The current selector isn't specific enough to override the default outline.

Reply:
The focus outline comes from `mixing/tab-focus.less`. Ideally we should just remove this file, but I'm not sure if there are templates that rely on it.